pkg/generate/app/cmd/newapp_test.go

```diff
@@ -53,6 +53,17 @@ func TestValidate(t *testing.T) {
 			env:                 map[string]string{"one": "first", "two": "second", "three": "third"},
 			parms:               map[string]string{},
 		},
+		"envfile": {
+			cfg: AppConfig{
+				GenerationInputs: GenerationInputs{
+					EnvironmentFile: "/tmp/envfile1",
+				},
+			},
+			componentValues:     []string{},
+			sourceRepoLocations: []string{},
+			env:                 map[string]string{"one": "first", "two": "second", "three": "third"},
+			parms:               map[string]string{},
+		},
```

Comment:
Thats a string that I am passing to envFileOperation. This is custom function that which creates or deletes the file with content in its definition - lines 27-39 and 179 containing the delete call. I will change this to a variable later. Need your feedback on the function. 

Also, I added another function in newapp.go which checks if the environment variable is in command line and won't overwrite it with value coming from file as suggested by @fabianofranz